client/src/components/Preview/Preview.js

```diff
@@ -125,45 +124,36 @@ class Preview extends Component {
 
     // Show iframe preview
     return (<iframe
-      style={{ visibility: this.state.frameLoaded ? 'visible' : 'hidden' }}
+      style={{ visibility: frameLoaded ? 'visible' : 'hidden' }}
       className="flexbox-area-grow preview__iframe"
       src={previewUrl}
-      onLoad={this.setFrameLoaded}
+      onLoad={setFrameLoadedState}
     />);
-  }
-
-  render() {
-    const { className, ViewModeComponent } = this.props;
-
-    const classNames = classnames('preview', className);
-    return (
-      <div className={classNames}>
-        {this.renderBody()}
-        <div className="toolbar toolbar--south">
-          <div className="btn-toolbar">
-            {this.buildToolbarButtons()}
-            <ViewModeComponent id="view-mode-toggle-in-preview-nb" area="preview" />
-            {this.renderMoreActions()}
-          </div>
+  };
+
+  const classNameValue = classnames('preview', className);
+  return (
+    <div className={classNameValue}>
+      {renderBody()}
+      <div className="toolbar toolbar--south">
+        <div className="btn-toolbar">
+          {buildToolbarButtons()}
+          <ViewModeComponent id="view-mode-toggle-in-preview-nb" area="preview" />
+          {renderMoreActions()}
         </div>
       </div>
-    );
-  }
-}
+    </div>
+  );
+};
 
 Preview.propTypes = {
   className: PropTypes.string,
   itemLinks: PropTypes.object,
   itemId: PropTypes.number,
-  onBack: PropTypes.func,
```

Comment:
This was unused, removed because eslint was complaining about it being in propTypes